Nhdr dev branch

NHDR-Quin opened the pull request:
Added valuesets from NHDR FHIR Mapping Team

nhdr-gianb reviewed (changes requested):
Some initial comments:
- Remove profiles/**PH_Period.fsh** as we have no use of a custom Period data type.
- Verify if vocabulary/**CS_CoverageCoPayTypeLocal** should be added in again instead of using the [existing](http://terminology.hl7.org/CodeSystem/coverage-copay-type) HL7 terminology service.
- Remove vocabulary/**CS_PatientType.fsh** and **VS_PatientType.fsh** since this is declared as **MemberType** in the release 01.

Since there are a lot of changes, indicate reason for addition and removal of other files to properly track changes. Such as:
    - CS_ConditionVerificationStatus
    - CS_ObservationMethod
    - VS_ConditionClinicalStatus
    - VS_DischargeDisposition
    - VS_EducationalAttainment
    - VS_EncounterHospitalizationDischargeDisposition
    - VS_EncounterLocationPhysicalType
    - VS_Gender
    - VS_HealthFacility
    - VS_MedicationStatus
    - VS_OccupationClassification
    - VS_ProcedureOutcome
    - VS_ServiceRequestStatus
    - and other noteable changes

Otherwise, this will result in the need to do a manual merge again, and/or re-review everything from top to bottom after a new merge.

NHDR-Quin commented:
Please use this as reference in Merging @nhdr-gianb. Thanks

- PH_Period.fsh - Please remove this file.
- CS_CoverageCoPayTypeLocal - retain this,  it is the additional local codes added in CoPayType Valueset.
- MemberType and PatientType - will be retain as we need to discuss this with the team, then we will update these valueset accordingly.

- CS_ConditionVerificationStatus - removed this, I directly bind to hl7 valueset
- CS_ObservationMethod - Retain, there are exclude changes in this valueset
- VS_ConditionClinicalStatus - Removed, it will be directly bind to hl7 valueset
- VS_DischargeDisposition -  Retain, it is additional valueset
- VS_EducationalAttainment - Retain, it is additional valueset
- VS_EncounterHospitalizationDischargeDisposition -Removed, changed the name to VS_DischargeDisposition
- VS_EncounterLocationPhysicalType - Removed, changed the name to VS_LocationPhysicalType
- VS_Gender - Retain, additional valueset for gender
- VS_HealthFacility - Retain, additional valueset bound to NHFR
- VS_MedicationStatus - Removed, already directed to valueset in alias
- VS_OccupationClassification - Retain, binded to PSOC
- VS_ProcedureOutcome - Retain, newly added valueset
- VS_ServiceRequestStatus - Retain, new added valueset

NHDR-Quin commented:
These items will be removed, while the remaining pushed files will be retained. @nhdr-gianb 

CS_ConditionVerficationStatus.fsh
CS_FDAProducts.fsh
CS_ObservationMethod.fsh
VS_ConditionClinicalStatus.fsh
VS_EncounterHospitalizationDischargeDisposition.fsh
VS_EncounterLocationPhysicalType.fsh
VS_EncounterServiceType.fsh
VS_MedicationCode.fsh
VS_MedicationStatus.fsh
VS_Occupation.fsh
